adapters/ucfunnel/ucfunnel_test.go

```diff
@@ -54,23 +55,43 @@ func TestMakeRequests(t *testing.T) {
 	}
 
 	var testCases = []struct {
-		in   []openrtb2.BidRequest
-		out1 [](int)
-		out2 [](bool)
+		giveRequest openrtb2.BidRequest
+		wantErr     bool
+		wantRequest bool
+		wantImpIDs  []string
 	}{
 		{
-			in:   []openrtb2.BidRequest{internalRequest01, internalRequest02, internalRequest03},
-			out1: [](int){1, 1, 0},
-			out2: [](bool){false, false, true},
+			giveRequest: internalRequest01,
+			wantErr:     true,
+			wantRequest: false,
+			wantImpIDs:  []string{},
+		},
+		{
+			giveRequest: internalRequest02,
+			wantErr:     true,
+			wantRequest: false,
+			wantImpIDs:  []string{imp.ID, imp2.ID, imp3.ID, imp4.ID, imp5.ID},
+		},
+		{
+			giveRequest: internalRequest03,
+			wantErr:     false,
+			wantRequest: true,
+			wantImpIDs:  []string{imp.ID, imp2.ID, imp3.ID, imp4.ID, imp5.ID},
 		},
 	}
 
-	for idx := range testCases {
-		for i := range testCases[idx].in {
-			RequestData, err := bidder.MakeRequests(&testCases[idx].in[i], nil)
-			if ((RequestData == nil) == testCases[idx].out2[i]) && (len(err) == testCases[idx].out1[i]) {
-				t.Errorf("actual = %v expected = %v", len(err), testCases[idx].out1[i])
-			}
+	for _, tc := range testCases {
+		RequestData, err := bidder.MakeRequests(&tc.giveRequest, nil)
+		if tc.wantErr == true {
```

Comment:
Nitpick: There is no need to include the `== true` part. Can simple be `if tc.wanterr {` and `if tc.wantRequest {`.

Reply:
done